app/package.json

```diff
@@ -10,13 +10,13 @@
     "postinstall": "npm run prisma-generate",
     "lint": "eslint --no-eslintrc --config .eslintrc.json .",
     "lint-fix": "eslint --no-eslintrc --config .eslintrc.json --fix .",
-    "test": "dotenv -e .env.test -- prisma db push && jest --runInBand",
+    "test": "dotenv -e .env.test -- prisma db push && node --experimental-vm-modules node_modules/.bin/jest --runInBand",
```

Comment:
since it is a good practice to reference tools directly from `node_modules` in NPM scripts, please use:
```
"test": "dotenv -e .env.test -- prisma db push && NODE_OPTIONS='--experimental-vm-modules' jest --runInBand",
```